src/Subscriber/PaginateElasticaQuerySubscriber.php

```diff
@@ -142,4 +142,17 @@ private function getRequest()
     {
         return $this->requestStack->getCurrentRequest();
     }
+
+    /**
+     * @param string $key
+     * @return mixed|null
+     */
+    private function getFromRequest($key)
```

Comment:
You may use type hints as PHP 7.1 is the minimal requirement.

Reply:
Yes, that true.